src/Common/src/Interop/Unix/System.Native/Interop.Stat.cs

```diff
@@ -15,6 +15,28 @@ internal static partial class Sys
         //
         // Explicitly saying Sequential disables that warning/error for consumers which only
         // use Stat in debug builds.
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct TimeSpec
+        {
+            internal long Seconds;
+            internal long Nanoseconds;
+
+            public static bool operator <(TimeSpec left, TimeSpec right)
+            {
+                if (left.Seconds < right.Seconds)
+                    return true;
+                if (right.Seconds < left.Seconds)
+                    return false;
+
+                return (left.Nanoseconds < right.Nanoseconds);
+            }
+
+            public static bool operator >(TimeSpec left, TimeSpec right)
```

Comment:
Is this really needed - can we get away with just `operator <` ?

Reply:
Compiler requires a matched pair for some reason.

Reply:
Then I would make a CompareTo method instead of the operator.